pelican/themes/notmyidea/templates/analytics.html

```diff
@@ -1,13 +1,11 @@
 {% if GOOGLE_ANALYTICS %}
     <script type="text/javascript">
-    var _gaq = _gaq || [];
-    _gaq.push(['_setAccount', '{{GOOGLE_ANALYTICS}}']);
-    _gaq.push(['_trackPageview']);
-    (function() {
-        var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-        ga.src = 'https://ssl.google-analytics.com/ga.js';
-        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-    })();
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+        ga('create', '{{GOOGLE_ANALYTICS}}', '{{GOOGLE_ANALYTICS_PROPERTY}}');
```

Comment:
I think the majority of people would only ever set their tracking ID, and would just leave the third element of the tuple as `auto`. Perhaps make this the default (and allow it to be overridden by a setting), or not have it configurable at all (ie. just hard-code the value of `auto` in)?